packages/scrubber-bar/src/scrubber-bar.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+@customElement('scrubber-bar')
```

Comment:
Suggestion: add `;` per house style

Reply:
Interesting.. so I added a semicolon and you can't actually use decorators with a semicolon. I got a `Parsing error: Declaration expected` when I added it.